.github/workflows/pull_request.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - uses: actions/setup-python@v4
         with:
           python-version: ${{ env.PYTHON_VERSION }}
+          cache: 'pipenv'
```

Comment:
Do we need our own `Cache virtualenv` set if we can use built in caching?.

Reply:
We can probably use built in caching but I didn't test it as it was cached the other way already.

Reply:
UPDATE: Removed all virtualenv install instances.